### PR TITLE
feat : added card and slider

### PR DESCRIPTION
**Pull request by Prashantrajput11**
added card :
Horizontal Card
image

Vertical Card and Dismiss Card
image

image

Card with shadow

image

Card with badge 

image

Card with text overlay
image

Card with text only

image

Added slider

image

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
+.hz--card-container {
+	display: flex;
+	background-color: var(--teritiary-color);
+	width: 600px;
```
Comment (shobit1337) [inline components/card.css]: you can use relative units like rems instead of px

**Review by shobit1337 — APPROVED** [review]
Rest all looks good to me. The design looks good!

File: components/card.css (hunk 1 above)
Comment (samyakshah3008) [inline components/card.css]: You can give max or min-width for responsiveness

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+/***********************/
+
+.vt--card-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	align-items: center;
+	background-color: var(--teritiary-color);
+	width: 300px;
+	height: 600px;
```
Comment (samyakshah3008) [inline components/card.css]: Please give in rem unit, it is good for responsiveness(refactor it later on)

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+	font-size: 2.2rem;
+}
+
+.vt--card-cta-btn {
+	text-decoration: none;
+	border-radius: 5px;
+	text-align: center;
+	font-size: 1.4rem;
+	/* font-size: 2.8rem; */
+}
```
Comment (samyakshah3008) [inline components/card.css]: You can remove this commented code later on for good code quality

File: components/card.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+	left: 90;
+	color: var(--primary-color);
+	background-color: rgba(0, 0, 0, 0.4);
+}
+
+/***********************/
+/***text only cards ***/
+/***********************/
+
+.card--text-only {
```
Comment (samyakshah3008) [inline components/card.css:186]: Liked how you have given comments, that helped for better review

File: components/card.css [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+	border: 2px solid white;
+	background-color: var(--primary-color);
```
Comment (samyakshah3008) [inline components/card.css]: You can give CSS variables

**Review by samyakshah3008 — APPROVED** [review]
Nice and clean UI 
Clean code with comments 

Given small suggestions in single comments, can refactor later on 

Rest is good for approval ✅

File: components/card.css (hunk 1 above)
Comment (Prashantrajput11) [reply components/card.css]: fixed

File: components/card.css (hunk 2 above)
Comment (Prashantrajput11) [reply components/card.css]: fixed

File: components/card.css (hunk 3 above)
Comment (Prashantrajput11) [reply components/card.css]: fixed

**Pull request merged**